src/css/global.css

```diff
@@ -40,59 +33,89 @@ p{
   --FONT_SIZE_20: 20px;
   --FONT_SIZE_24: 24px;
   --FONT_SIZE_30: 30px;
+  --FONT_SIZE_36: 36px;
+  --FONT_SIZE_44: 44px;
   --FONT_SIZE_48: 48px;
   --FONT_SIZE_50: 50px;
   --FONT_SIZE_72: 72px;
-  --FONT_SIZE_44: 44px;
 
   --FONT_WEIGHT_700: 700;
 }
 
-@font-face {
+html {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+p, a, input, button, span {
   font-family: 'Axiforma';
-  src: url(../fonts/Axiforma-Regular.ttf);
-  font-display: swap;
-  font-style: normal;
+  letter-spacing: 0.02em;
```

Comment:
Why em?

Reply:
I didn’t know how more convenient, I changed it to px